benchmarks/chained-function-serving/go.mod

```diff
@@ -1,32 +1,49 @@
 module tests/chained-functions-serving
 
-go 1.16
+go 1.17
```

Comment:
if we update Golang, why not go to 1.18 directly?

Reply:
I did the next version bump up as xdt had to be updated from 1.16 as semconv dependency was giving some errors.

Reply:
go to 18 then

Reply:
Ok. Will do.

Reply:
A lot of new files will need a change for this. I suggest we do this in a new PR.

Reply:
ok, but please either do it consistently and quickly or open an Issue with an enumeration what needs to be upgraded. vSwarm is so big

Reply:
So I think it might be ok to not change all of them.
In my understanding at lest this go version only states the minimum required version of go. So if the mod file states go 1.16 then you can build the module with any version 1.16+
We would only need to update this version number if the code changes.

Reply:
However, I agree with @ustiugov that we should update all mod files if we change the golang base image as we then tests with 1.18. We would not recognize if things break for a lower version even though the mod file states it

Reply:
Shall I update the version for tools also or benchmarks only?